package.json

```diff
@@ -30,8 +30,8 @@
   "peerDependencies": {
     "react": ">=16.13.1",
     "react-dom": ">=16.13.1",
-    "styled-system": ">= 5",
-    "styled-components": ">= 5"
+    "styled-components": ">= 5",
```

Comment:
Essas changes acredito que pegaram porque essa branch está atrás da develop, acho que vamos ter que fazer um rebase. Pode chamar a mim ou um dos guris que a gente te dá uma força pra fazer.